Documentation/operations/upgrade.rst

```diff
@@ -298,6 +298,14 @@ Removed Options
 
 * The previously deprecated high-scale mode for ipcache has been removed.
 
+Helm Options
+~~~~~~~~~~~~
+
+* The Helm options ``hubble.export.fileMaxSizeMb``, ``hubble.export.fileMaxBackups``
+  and ``hubble.export.fileCompress`` have been deprecated in favor of their corresponding exporter
+  type options under (``hubble.export.static``, ``hubble.export.dynamic``) and will be removed in
```

Comment:
Is that enough to help users find the new options? For the dynamic exporter in particular, it's also under `.content`? But then I haven't used these myself so it could be just me.

Reply:
I assumed looking at the values file would be enough to map the fields (they have the same name), but overall I agree we could be more descriptive with these, let me update.

Reply:
I've updated the formulation, hopefully this is clearer.